src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -173,23 +173,13 @@
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
         (let [count-all (count submissions)
-              success-count (->> submissions (filter all-success?) count)
-              running-count (->> submissions (filter any-running?) count)
-              fail-count (->> submissions (filter any-failed?) count)
-              unknown-count (- count-all success-count running-count fail-count)]
+              grouped-by-status (group-by #(% "status") submissions)]
```

Comment:
This is correct.  The function, fully written out, would be:

    (fn [submission] (get submission "status"))

Data structures can act as their own getter function, so the `get` is unnecessary:

    (fn [submission] (submission "status"))

Then we use the function macro since it's a function of one variable that's only referenced once:

    #(% "status")